modules/mesaides/manifests/monitor_config.pp

```diff
@@ -0,0 +1,36 @@
+define mesaides::monitor_config () {
```

Comment:
Why call this `monitor_config` rather than simply `monitor`? This also starts the service, and we are in a configuration management system, so somehow everything in that scope is always a “config”, isn't it? 🙂